### DISCUSSION TITLE
Why wgsl do not support addition assignment?

### DISCUSSION THREAD

**Discussion by wewindy** [gpuweb/gpuweb#2598] [Q&A] [answered]
As the title.

**Comment by Kangz** [comment, marked as answer]
It does, see the [compound assignment](https://gpuweb.github.io/gpuweb/wgsl/#compound-assignment-sec) section. It is just that implementations haven't caught up with the spec yet. (For example implementation in Chromium is tracked in [this issue](https://bugs.chromium.org/p/tint/issues/detail?id=1325&q=jrprice&can=2).)

**Reply by wewindy** [reply]
tks!
